subscription_integration_test.go

```diff
@@ -195,8 +199,7 @@ func TestSubscriptionAllFieldsWithBillingDayOfMonth(t *testing.T) {
 	}
 
 	// Update
-	sub2, err := g.Update(ctx, &SubscriptionRequest{
-		Id:     sub1.Id,
+	sub2, err := g.Update(ctx, sub1.Id, &SubscriptionRequest{
 		PlanId: "test_plan_2",
```

Comment:
Can you add an expectation for this update that checks that the ID of the audition returned starts the same? There's already an expectation to check if the ID is not an empty string so that just needs changing to check if it requires the original ID.

Reply:
Where is that? I see this: https://github.com/lionelbarrow/braintree-go/pull/243/files#diff-fe0b52236b5a897ed31d1dac80a712b6R283

Reply:
Ahh I see it now, here https://github.com/vania-pooh/braintree-go/blob/945a8238b432278e9d47aab6267fe04614083ac3/subscription_integration_test.go#L216.